### PR TITLE
Document the hardware integration

### PR DESCRIPTION
**Pull request by emontnemery**
## Proposed change

Document the hardware integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by balloob** [comment]
wrong branch 🙃

File: source/_integrations/hardware.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ha_integration_type: integration
+---
+
+The Hardware integration provides an API which offers information about certain board types, USB dongles, and CPU and memory usage.
+
+This integration is by default enabled, unless you've disabled or removed the [`default_config:`](/integrations/default_config/) line from your configuration. If that is the case, the following example shows you how to enable this integration manually:
+
+```yaml
+# Example configuration.yaml entry
+system_health:
```
Comment (MartinHjelmare) [inline source/_integrations/hardware.markdown]: Is this correct?

File: source/_integrations/hardware.markdown [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: Hardware
+ha_category:
+  - Other
+ha_release: 2022.7
```
Comment (MartinHjelmare) [inline source/_integrations/hardware.markdown]: Shouldn't this go to current if it's already been released?

Comment (emontnemery) [reply source/_integrations/hardware.markdown]: Oh, you're right 🤦
Fixed now.

File: source/_integrations/hardware.markdown (hunk 1 above)
Comment (emontnemery) [reply source/_integrations/hardware.markdown]: Nope, fixed now

**Review by janiversen — COMMENTED** [review]
LGTM.

Except I am not sure why I was requested as reviewer, so I will leave hitting the approve button to people that matters 😃

File: source/_integrations/hardware.markdown (hunk 1 above)
Comment (MartinHjelmare) [reply source/_integrations/hardware.markdown]: It's not fixed yet. :wink:

Comment (emontnemery) [reply source/_integrations/hardware.markdown]: Oops, now it should be fixed 🙃

File: source/_integrations/hardware.markdown [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  - Other
+ha_release: 2022.6
+ha_quality_scale: internal
+ha_codeowners:
+  - '@home-assistant/core'
+ha_domain: hardware
+ha_integration_type: integration
+---
+
+The Hardware integration provides an API which offers information about certain board types, USB dongles, and CPU and memory usage.
```
Comment (ludeeus) [inline source/_integrations/hardware.markdown]:
```suggestion
The Hardware integration provides an API which offers information about certain board types, USB dongles, and resource usage.
```
"and x and y" does not read well 🤷‍♂️

**Review by MartinHjelmare — DISMISSED** [review]

**Review by ludeeus — APPROVED** [review]

**Pull request merged**